sql-db/hibernate-reactive/src/test/resources/test.properties

```diff
@@ -0,0 +1,2 @@
+ts.db2.container.privileged-mode=true
+ts.db2.startup.timeout=10m
```

Comment:
why these props are needed?

Reply:
Db2 requires, that container is started in privileged mode and it has a long start-up phase, so we use a long possible timeout.